bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/aQGC_WZA_LO/WZA_WToLNu_LOaQGC_run_card.dat

```diff
@@ -0,0 +1,267 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update full_run_card                                          *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+#*********************************************************************
+  100000 = nevents ! Number of unweighted events requested 
```

Comment:
line30, this number will not be used, but to be not confusing, please change it to 3000

Reply:
> line30, this number will not be used, but to be not confusing, please change it to 3000

I modified it 100000 -> 3000.
Thank you very much.